### DISCUSSION TITLE
PCL2 的任务栏小按钮

### DISCUSSION THREAD

**Discussion by tangge233** [Meloong-Git/PCL#4016] [讨论]
在 PCL 1 时代，任务栏里会有一个启动游戏的小按钮
![image](https://github.com/Hex-Dragon/PCL2/assets/50769997/9d10e93a-0ad0-42a8-86d6-29d449d3cdc6)

那么 PCL 2 时代，还有必要加回这样的小功能吗？👀
![image](https://github.com/Hex-Dragon/PCL2/assets/50769997/92f7f210-8870-404c-adee-8e62826a3f71)

**Comment by MoYuan-CN** [comment]
不是很好弄好像？
我暂时没有找到类似反馈，或许你可以开一个新的反馈

**Comment by WorldHim** [comment]
感谢反馈 帮你交了 #4029
